utils/common/src/main/java/brooklyn/util/javalang/Threads.java

```diff
@@ -44,7 +46,15 @@ public static boolean removeShutdownHook(Thread hook) {
             return Runtime.getRuntime().removeShutdownHook(hook);
         } catch (IllegalStateException e) {
             // probably shutdown in progress
-            log.debug("cannot remove shutdown hook "+hook+": "+e);
+            String text = Exceptions.collapseText(e);
+            if (text.contains("Shutdown in progress")) {
```

Comment:
This is a good candidate for a typed exception. Always wondered why the general dislike to typed exceptions in Brooklyn?